domain/schema/patch.go

```diff
@@ -26,25 +39,39 @@ func readPatches(entries []fs.DirEntry, fs fs.ReadFileFS, fileName func(string)
 
 	slices.Sort(names)
 
-	var (
-		patches     []func() schema.Patch
-		postPatches []func() schema.Patch
-	)
+	var patches []func() schema.Patch
 	for _, name := range names {
-		data, err := fs.ReadFile(fileName(name))
+		if strings.HasSuffix(name, postPatchFileSuffix) {
+			continue
+		}
+
+		data, err := fs.ReadFile(filepath.Join(baseDir, name))
 		if err != nil {
-			panic(err)
+			return nil, err
 		}
 
 		fn := func() schema.Patch {
 			return schema.MakePatch(string(data))
 		}
 
-		if strings.HasSuffix(name, pathFileSuffix) {
-			postPatches = append(postPatches, fn)
-			continue
-		}
 		patches = append(patches, fn)
 	}
-	return patches, postPatches
+	return patches, nil
+}
+
+func readPostPatches(fs fs.ReadFileFS, baseDir string, postPatchFiles []string) ([]func() schema.Patch, error) {
+	patches := make([]func() schema.Patch, len(postPatchFiles))
```

Comment:
I kinda think we should sort the postPatchFile names here, so it's the same as readPatches method. It would prevent potential footguns if someone puts them in the wrong order?

Reply:
Either way includes food guns I think. Someone way forgot to prefix a number, or try and get fancy, assuming that the order they pick up from `(controller|model)PostPatchFilesByVersion` is respected